libr/arch/p/m680x_cs/plugin.c

```diff
@@ -515,6 +515,25 @@ static bool decode(RArchSession *as, RAnalOp *op, RAnalOpMask mask) {
 					insn->op_str[0]?" ": "", insn->op_str);
 			r_str_replace_in (op->mnemonic, strlen (op->mnemonic) + 1,
 				"ptr ", "", true);
+			// we don't really output operands in intel order but we can clean up
+			// XXX: this is not finished
+			if (as->config->syntax == R_ARCH_SYNTAX_INTEL) {
+				op->mnemonic = r_str_replace (op->mnemonic,
```

Comment:
probably better to move this logic into a separate function, so op-> is only referenced once and not 10 times

Reply:
Indeed, and it's easier on the indentation.